open-company/open-company-web
# Fix org name

Card: https://trello.com/c/scILHe01

To test:
- signup for a new org via email, use a short name for the org for now
- finish onboard and get to AP
- [ ] do you see the Carrot logo in top left corner
- [ ] do you also see the org name you choose?
- now copy the public url of a post
- [ ] do you see the carrot logo and your org name in top left corner? looks good?
- go back to AP
- open org settings
- [ ] do you see the green plus and no carrot logo?
- now upload a new logo for the company
- refresh the public post page
- [ ] do you see the correct avatar and name?
- now onboard a new org and use a very long name for it (the maximum length you can in the field)
- [ ] repeat the steps above and make sure everything is working fine
- now invite a user already active in another org to this new org you just created
- logout and log back in with the newly invited user (to refresh the JWT and see the new org)
- [ ] do you see a dropdown arrow on the right of you name?
- [ ] is the dropdown arrow at the end of the org name w/o too much white space btw the 2?
- click it
- [ ] the orgs dropdown looks good?
- switch org
- [ ] everything good here too?
- now with the org with a long name go to org settings
- now change org avatar
- [ ] do you see the new avatar in the top left corner and also in the orgs dropdown?
